packages/tlschema/src/misc/TLRichText.ts

```diff
@@ -0,0 +1,34 @@
+import { T } from '@tldraw/validate'
+
+/** @public */
+export const richTextValidator = T.object({ type: T.string, content: T.arrayOf(T.unknown) })
+
+/** @public */
+export type TLRichText = T.TypeOf<typeof richTextValidator>
+
+/** @public */
+export function convertTextToTipTapDocument(text: string): TLRichText {
```

Comment:
i normally wouldn't suggest this, but as this function is called so often and in so many examples etc, is there a shorter name that would still make sense? would just `toRichText` work?